lit_gpt/config.py

```diff
@@ -27,6 +27,8 @@ class Config:
     rotary_percentage: float = 0.25
     parallel_residual: bool = True
     bias: bool = True
+    # just for Qwen
+    is_Qwen: Optional[bool] = None
```

Comment:
I think we should avoid this in favor of something that characterizes Qwen, like having bias only in `c_attn`.
For the time being we could rename this as `attn_bias`, and then in the future turn `bias` into a `Option[bool, List[str]]` if there's a need for it.